low-level/requirements.txt

```diff
@@ -1,10 +1,11 @@
-jsonschema==3.1.1
+jsonschema==3.2.0
 pika==1.1.0
-pyserial==3.4
+pyserial==3.5
 python-daemon==2.2.4
-requests==2.22.0
-zope.component==4.6
-zope.event==4.4
-zope.interface==4.7.1
-pyinotify==0.9.4
+requests==2.25.1
+zope.component==4.6.2
+zope.event==4.5
+zope.interface==5.2.0
+pyinotify==0.9.6
 python-consul==1.1.0
+paramiko==2.7.2
```

Comment:
paramiko and requests are used cortx wide i believe. Confirm with @shailesh-vaidya and if yes, remove from here

Reply:
@gargpalak048 as discussed in scrum, packages we would remove from requirements.txt is fine where cortx-provisioner would do the stack deployment. But in env, where sspl would be self-provisioning we would need these packages still, so install these as part of sspl-dev-deploy script.

Reply:
@cdeshmukh I have added the code in sspl_dev_deploy script to install paramiko & requests module.
I have kept the code in the comment block.
Once @shailesh-vaidya will confirm, we can remove the comments and can remove the pkgs from requirement.txt file.

Reply:
Comment out from here and enable in dev deploy anyways, that is more safer, without affecting other components

Reply:
Done the changes.